src/Ng/Arguments.cs

```diff
@@ -146,6 +146,11 @@ public static class Arguments
         /// If the queue contains more messages than this, the job will not requeue any invalid packages.
         /// </summary>
         public const string MaxRequeueQueueSize = "maxRequeueQueueSize";
+
+        /// <summary>
+        /// If true, packages are expected to have at least a repository signature.
+        /// </summary>
+        public const string ExpectsSignature = "expectsSignature";
```

Comment:
Suggestion:  `expectSignature` or `requireSignature` throughout the PR.

Reply:
Done